lasagne/layers/conv.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, incoming, num_filters, filter_size, stride=1, pad=0,
         self.num_filters = num_filters
         self.filter_size = as_tuple(filter_size, n, int)
         self.flip_filters = flip_filters
-        self.stride = as_tuple(stride, n, int)
+        self.stride = as_tuple(stride, n)
```

Comment:
OK. I actually think that removing the `np.random.randInt` already fixed that issue, I pushed a commit to go back to having `int` in the `as_tuple`.